### PR TITLE
Update template branding

### PR DESCRIPTION
**Pull request by dreddy-work**
Update template point to .NET8.0

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7595)

### CODE REVIEW

**Review by RussKie — APPROVED** [review]

**Comment by RussKie** [comment]
Probably a good idea to update this too:
https://github.com/dotnet/winforms/blob/3a7abc5587a879b9a75a7f1e0538bc42b2ccccc6/pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj#L5

**Comment by dreddy-work** [comment]
> Probably a good idea to update this too:
> 
> https://github.com/dotnet/winforms/blob/3a7abc5587a879b9a75a7f1e0538bc42b2ccccc6/pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/Microsoft.Dotnet.Winforms.ProjectTemplates.csproj#L5

let's do that separately after 7.0 is GA'd?

File: pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/content/WinFormsLibrary-VisualBasic/.template.config/template.json [hunk 1]
```diff
@@ -44,15 +44,20 @@
           "description": "Target net6.0",
           "displayName": ".NET 6.0"
         },
+        {
+          "choice": "net7.0",
```
Comment (Tanya-Solyanik) [inline pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/content/WinFormsLibrary-VisualBasic/.template.config/template.json:48]: .NET 7 option is now duplicated.

Comment (dreddy-work) [reply pkg/Microsoft.Dotnet.WinForms.ProjectTemplates/content/WinFormsLibrary-VisualBasic/.template.config/template.json:48]: thanks @Tanya-Solyanik for catching this.

**Pull request merged**